src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/configs/decider/WorkLoadTypeConfig.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.configs.decider;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.core.Config;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.core.NestedConfig;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Predicate;
+
+/**
+ * "workload-type": {
+ *   "priority-order": ["ingest", "search"]
+ * }
+ */
+public class WorkLoadTypeConfig {
+
+  private static final String PRIORITY_ORDER_CONFIG_NAME = "priority-order";
+  private static String INGEST = "ingest";
+  private static String SEARCH = "search";
+  public static final List<String> DEFAULT_PRIORITY_ORDER = Collections.unmodifiableList(
+      Arrays.asList(INGEST, SEARCH));
+  private Config<List<String>> priorityOrder;
+  private Predicate<List<String>> listValidator;
+
+  public WorkLoadTypeConfig(NestedConfig configs) {
+    listValidator = (list) -> {
```

Comment:
Since there is only search and ingest (2 values), we can make this simpler by renaming the config. Instead of taking a list, we can do things like:
```
{ "decider-config-settings": {
    "prefer-ingest-over-search": true
    }
}
```

Reply:
done. changed the config to 
"workload-type": {
      "prefer-ingest": true,
      "prefer-search": false
    },

This will allow user to not assign workload preference to any queue type